operator/ARCHITECTURE.md

```diff
@@ -54,31 +54,31 @@ of Pilot replicas is a component setting, because it refers to a component which
 cluster. Most K8s platform settings are necessarily component settings.
 The available features and the components that comprise each feature are as follows:
 
-| Feature | Components |
-|---------|------------|
-CRDs, and other cluster wide configs | Base
-Traffic Management | Pilot
-Security | Pilot
-Configuration management | Pilot
-AutoInjection | Pilot
-Gateways | Ingress gateway
-Gateways | Egress gateway
-Policy | Policy (deprecated)
-Telemetry | Telemetry (deprecated)
+| Feature                              | Components             |
```

Comment:
FYI, the operator package lacks maintenance, so it's not worth spending efforts on this package.